lib/bokken/uploaders/emblem.ex

```diff
@@ -4,16 +4,19 @@ defmodule Bokken.Uploaders.Emblem do
   """
   use Waffle.Definition
   use Waffle.Ecto.Definition
+  alias Bokken.Documents.File
 
   @versions [:original]
   @extension_whitelist ~w(.jpg .jpeg .gif .png)
+  @max_file_size 5_000_000
```

Comment:
```suggestion
  @max_file_size 1_000_000
```

1MB is enough for this. These are uploaded by us, so that, we can control it 🙏🏼